app/src/main/java/mozilla/components/lib/publicsuffixlist/ext/ByteArray.kt

```diff
@@ -15,7 +15,7 @@ private const val BITMASK = 0xff.toByte()
  * Performs a binary search for the provided [labels] on the [ByteArray]'s data.
  *
  * This algorithm is based on OkHttp's PublicSuffixDatabase class:
- * https://github.com/square/okhttp/blob/master/okhttp/src/main/java/okhttp3/internal/publicsuffix/PublicSuffixDatabase.java
+ * https://github.com/square/okhttp/blob/1977136/okhttp/src/main/kotlin/okhttp3/internal/publicsuffix/PublicSuffixDatabase.kt
```

Comment:
The original link is dead now, but I'm not sure whether changing this here is the right way to go. We could submit the change upstream.

Reply:
Yeah I'll do that.